aa-software2112/SOEN_344
# Frontend/make appointment nurse #96

Implemented both frontend and backend for make appointment nurse [#96]. 

First time using Vue for this kind of logic so please let me know what I could improve on/fix - there are a few things I'm definitely iffy about and would like to refactor for Milestone 2 but as of now it's working for me

To test:
- A nurse should be able to book an appointment after choosing a patient.
- A nurse should NOT be able to book an appointment without choosing a patient.
- A nurse should be able to search for availabilities after choosing a patient AND a doctor and should only see appointments belonging to that doctor.
